kafka/client_async.py

```diff
@@ -200,19 +200,21 @@ def __init__(self, **configs):
         for key in self.config:
             if key in configs:
                 self.config[key] = configs[key]
-
+        self._closed = False
+        self._wake_r, self._wake_w = socket.socketpair()
+        self._selector = self.config['selector']()
```

Comment:
It'd be good to have a comment here noting that the above properties should be top so as to not break when `__del__` is called.

Reply:
sure !!!  done.